content/en/service_management/workflows/private_actions.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Private actions
+kind: Documentation
+disable_toc: false
+further_reading:
+- link: "service_management/app_builder/connections"
+  tag: "Documentation"
+  text: "App Builder Connections"
+- link: "service_management/workflows/connections"
+  tag: "Documentation"
+  text: "Workflow Connections"
+---
+
+{{< callout url="https://google.com" btn_hidden="false" header="Join the Beta!">}}
+Private Actions are in beta. Use this form to request access today.
+{{< /callout >}}
+
+## Overview
+
+Private actions allow your Datadog workflows and apps to interact with services hosted on your private network without opening a port in your firewall. To use private actions, you must install a private action runner on a host in your network and pair the runner with a Datadog Connection.
+
+The following integrations support private actions:
+- [Kubernetes][1]
+- [Postgres][2]
+- [Jenkins][3]
+- [Temporal][4]
+- The [HTTP][5] action
+
+## Prerequisites
+
+The private action runner requires a Linux host that is able to reach any internal services you want to call from an action or app. The host has the following requirements:
+- 2GB of RAM
+- Docker (with Docker Compose if that is your preference)
+- App Builder connections require SSL termination
```

Comment:
this should be more like

> To use App Builder with Private Actions, you must be able to point a hostname to the runner container. This hostname must be resolvable by your App users. It will be called using HTTPS so you must bring your own SSL termination as well.

This is probably the most complex part of the instructions so I'm hoping customers will have feedback on how to improve it!

same change applies top the app builder version of this page